Scripts/developer_scripts/cgal_demo_copy_all_dlls_cygwin.sh

```diff
@@ -3,9 +3,13 @@
 #use this script from inside the build directory of the Polyhedron demo
 #Needs the Qt5_DIR env variable set to <Qt5_ROOT>/lib/cmake/Qt5
 
-declare config="Release"
+#No config : in autotest_cgal we use NMake as generator
+#If using MSVC Generator, declare config="Release"
+
+
+declare config=""
```

Comment:
Then why not take config as an extra argument?
```suggestion
#No config : in autotest_cgal we use NMake as generator
#If using MSVC Generator, declare config="Release"


declare config="$2"
```